synth.py

```diff
@@ -34,96 +34,50 @@
     version="v3",
     bazel_target="//google/monitoring/v3:monitoring-v3-py",
     include_protos=True,
+    proto_output_path="google/cloud/monitoring_v3/proto"
```

Comment:
I'm a little confused by this. We shouldn't be generating the regular python protos if we're using the microgenerator. Is there something else that needs them, or am I misinterpreting what this line does?

Reply:
It just copies over the *.proto files, it doesn't generate any python protos.

Reply:
Okay, good to know. Is this a backwards compatibility thing?

Reply:
This is more like informational. The old client libs contains all the *.proto files so in the new one I just keep them.